worker/src/api/index.ts

```diff
@@ -116,7 +118,50 @@ router
     }
   });
 
-router.use("/emojis", emojis);
+router
+  .use(
+    basicAuth({
+      users: { admin: env.LANGFUSE_WORKER_PASSWORD },
+    })
+  )
+  .post("/ingestion", async (req, res) => {
+    try {
+      if (!redis) {
+        throw new Error("Redis connection not available");
+      }
+
+      const { body } = req;
+      const events = ingestionApiSchemaWithProjectId.safeParse(body);
+
+      if (!events.success) {
+        logger.error(events.error, "Failed to parse ingestion event");
+
+        return res.status(400).json(events.error);
+      }
+
+      const { batch, projectId } = events.data;
+
+      if (!ingestionFlushQueue) {
+        throw Error("Ingestion flush queue not available");
+      }
+
+      await new IngestionService(
```

Comment:
Before this becomes the ingestion pipeline, we will need to add auth. Created a ticket.